RecoTracker/LSTCore/src/alpaka/Event.h

```diff
@@ -0,0 +1,209 @@
+#ifndef RecoTracker_LSTCore_src_alpaka_Event_h
+#define RecoTracker_LSTCore_src_alpaka_Event_h
+
+#include "RecoTracker/LSTCore/interface/alpaka/Constants.h"
+#include "RecoTracker/LSTCore/interface/alpaka/LST.h"
+#include "RecoTracker/LSTCore/interface/Module.h"
+
+#include "Hit.h"
+#include "Segment.h"
+#include "Triplet.h"
+#include "Kernels.h"
+#include "Quintuplet.h"
+#include "MiniDoublet.h"
+#include "PixelQuintuplet.h"
+#include "PixelTriplet.h"
+#include "TrackCandidate.h"
+
+#include "HeterogeneousCore/AlpakaInterface/interface/host.h"
+
+using ::lst::EndcapGeometryBuffer;
+using ::lst::LSTESData;
+using ::lst::ModulesBuffer;
+using ::lst::PixelMap;
```

Comment:
[trivial]

Can you move these inside the `ALPAKA_ACCELERATOR_NAMESPACE::lst` ?
Potentially, they could be simplified to
```c++
namespace ALPAKA_ACCELERATOR_NAMESPACE {
  namespace lst {

    using namespace ::lst;
    ...
```